### ISSUE TITLE
FW rules - why?

### ISSUE THREAD

**Issue by jaromirk** [nmdange2/HyperV_DSC#5]
All these WMI/Remote Event Log management/ping... rules are enabled automatically when Cluster role is installed. Thus it's duplicite.

https://github.com/nmdange2/HyperV_DSC/blob/master/DSC/HyperVFabricSettings.ps1#L150 ...

**Comment by nmdange2** [comment]
Yes although I do also use these settings on standalone Hyper-V hosts that don't have clustering installed.  Since it's in the DSC config, it can also ensure the rules don't get disabled by accident.

**Comment by jaromirk** [comment]
Got it... I prefer limit access to certain people (the one that do click stuff... that's why I dont enable RDP in WSLab :-D )
